README.md

```diff
@@ -2,95 +2,147 @@
-communicate directly with Astra. They have excellent support for Astra out-of-the-box. To use a
-DataStax driver with Astra follow [this][driver-guide] guide.
+1. Generate a token following the Astra [instructions](https://docs.datastax.com/en/astra/docs/manage-application-tokens.html#_create_application_token). This step will display your Client ID, Client Secret, and Token; make sure you download the information for the next steps. Store the secure bundle in `/tmp/scb.zip` to match the example below.
 
-## Configuration
+2. Create `cql-proxy.yaml`. You'll need to add three sets of information: arguments, volume mounts, and volumes.
 
-To pass configuration to the cql-proxy both command line flags and environment variables can be used. Below are examples of
-the same command using both methods
+ - Argument: Modify the local bundle location, username and password, using the client ID and client secret obtained in the last step to the container argument.   
 
-Flags
+      ```
+      command: ["./cql-proxy"]
+      args: ["--bundle=/tmp/scb.zip","--username=Client ID","--password=Client Secret"]
+      ```
 
-```sh
-docker run -v <your-secure-connect-bundle.zip>:/tmp/scb.zip -p 9042:9042 \
-  --rm datastax/cql-proxy:v0.0.2 \
-  --bundle /tmp/scb.zip --username <astra-client-id> --password <astra-client-secret>
-```
+- Volume mounts: Modify `/tmp/` as a volume mount as required.
 
-Environment Variables
+      volumeMounts:
+        - name: my-cm-vol
+        mountPath: /tmp/
+ 
 
-```sh
-docker run -v <your-secure-connect-bundle.zip>:/tmp/scb.zip -p 9042:9042  \
-  --rm datastax/cql-proxy:v0.0.2 \
-  -e BUNDLE=/tmp/scb.zip -e USERNAME=<astra-client-id> -e PASSWORD=<astra-client-secret>
-```
+- Volume: Modify the `configMap` filename as required. In this example, it is named `cql-proxy-configmap`. Use the same name for the `volumes` that you used for the `volumeMounts`. 
+
+      volumes:
+        - name: my-cm-vol
+          configMap:
+            name: cql-proxy-configmap        
+    
+3. Create a configmap. Use the same secure bundle that was specified in the `cql-proxy.yaml`.
+      
+      ```sh
+      kubectl create configmap cql-proxy-configmap --from-file /tmp/scb.zip 
+      ```
+
+4. Check the configmap that was created. 
+
+    ```sh
+    kubectl describe configmap config
+      
+      Name:         config
+      Namespace:    default
+      Labels:       <none>
+      Annotations:  <none>
+
+      Data
+      ====
+
+      BinaryData
+      ====
+      scb.zip: 12311 bytes
+    ```
+
+5. Create a k8ssandra deployment with the YAML file you created:
+
+     ```sh
+     kubectl create -f cql-proxy.yaml
+     ```
 
-To see what options are available the `-h` flag will display a help message listing all flags and their corresponding descriptions
-and environment variables
+6. Check the logs:
+    ```sh
+    kubectl logs <deployment-name>
+    ```
+
+## Configuration
```

Comment:
This should be higher up (not below the k8s config, if that's to remain in the main README).

Reply:
moved it.